### ISSUE TITLE
Edit user profile image as admin + change the category image upload (Chrome)

### ISSUE THREAD

**Issue by robjens** [NodeBB/NodeBB#4247]
### TL;DR;

1) Can remove a user profile picture as admin but doing so **without save** will at least detroy the option from the list of images available and not bring it back, but maintain that image (dead link) in the profile itself (it doesn't seem to save-save but destructive on the selection list item anyway).
2) Should I be able to add profile pictures (as admin) of other users? If so bug. If not, is delete even supposed to be possible then? A bit unclear.

#### Longer version:

While troubleshooting category 404's in the admin, one of the things I had to do was to replace that of all the users (1 in this case, of 4 users 2 had avatars default and 2 had custom pictures, both were destroyed by what I presume was the latest upgrade but cannot validate that).

Now since I had my own replaced already but this one giving 404 on the other user (by means of the permissions list users showing profile pictures), I went to utilize my special admin powers and went to the public area and his profile. I noticed a bug there.

User profile -> Edit -> Options (below dead link image) -> Edit picture

Opens a dialog window. List has 2 items, 1 default avatar (not user) and 2 the dead image. Clicking remove image works as expected. Closing the dialog **without pressing save** closes the window. Profile image remains dead link (expected). Click edit profile picture again and dialog opens again. Only shows 1 default avatar (unexpected). Click avatar and do **not save**, close the window. Image remains dead link (expected). Open the dialog again, select avatar now **save** and close. Changes the image to avatar (expected). Now I thought, let's go do an update of his profile picture

![image](https://cloud.githubusercontent.com/assets/11848802/13224953/bbb5017c-d982-11e5-9e03-d82ef5aed86f.png)

No way to add a picture (unexpected but might be intended? Not sure).

Speaking of images and uploading, I got a warning in Chrome when uploading a new category image:

![image](https://cloud.githubusercontent.com/assets/11848802/13225007/f53215ac-d982-11e5-8da6-222cdc6f0a4b.png)

I thought I could drop that in here as well since it is such a minor one.

**Comment by robjens** [comment]
Minor note/suggestion/request... could we use something like e.g. mousetrap to have keys work on e.g. dialogs such as the one above? I really (and I'm sure many more) am used to pressing `ESC` to close a popup window (as form of cancel)

**Comment by pichalite** [comment]
@robjens did you try it with the default theme (persona)? This may be an issue with the theme you are using.

**Comment by pichalite** [comment]
Also make sure you have these options enabled in ACP -> Settings -> Uploads

![image](https://cloud.githubusercontent.com/assets/10014774/13228966/68fd6d36-d953-11e5-928c-5d6cf7ddfd52.png)


![image](https://cloud.githubusercontent.com/assets/10014774/13228956/5e5d9f7c-d953-11e5-8801-676d7c83c544.png)

**Comment by robjens** [comment]
@pichalite I will try with the default theme, my bad. I believe this way the Google Material design and might even be slightly tweaked already so I'll see if the behavior persists. Anyway I presume it doesn't, since I'm inclined to think you are unable to reproduce this.

Aaaanddd... just checked. Behaviour is gone (just did a update, Material Theme was one of it) so it must have been fixed then.

**Comment by psychobunny** [comment]
Thanks picha
